slash/react/src/Form/Select/__tests__/SelectInput.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { SelectInput } from "../SelectInput";
+
+const options = [
+  { value: "fun", label: "For fun" },
+  { value: "work", label: "For work" },
+  { value: "drink", label: "For drink" },
+];
+
+describe("SelectInput", () => {
+  it("should have label", async () => {
+    // Act
+    render(
+      <SelectInput
+        label="label select input"
+        mode="default"
+        options={options}
+      />,
+    );
+
+    // Assert
+    const labelSelectInput = screen.getByText("label select input");
+    expect(labelSelectInput).toHaveClass("af-form__group-label");
+    expect(
+      screen.getByRole("combobox", { name: "label select input" }),
+    ).toBeInTheDocument();
```

Comment:
Je pensais plus à ça :
```suggestion
    const labelSelectInput = screen.getByRole("combobox", { name: "label select input" });
    expect(labelSelectInput).toHaveClass("af-form__group-label");
```